ui-tests/src/test/java/org/jenkins/plugins/pitmutation/PitMutationTest.java

```diff
@@ -0,0 +1,399 @@
+     */
+    @Test
+    public void verifyTestExamined() {
+        MutationDetailView classView = getClassView();
+
+        List<String> tests = classView.getTestsExamined();
+
+        assertThat(tests.size()).isEqualTo(21);
+        assertThat(tests.get(0)).contains("edu.hm.hafner.util.PathUtilTest", "shouldReturnFallbackOnError",
+            "[test-template-invocation:#3] (8 ms)");
+    }
+
+    /**
+     * Checks if the mutators are displayed correctly.
+     */
+    @Test
+    public void verifyMutators() {
+        MutationDetailView classView = getClassView();
+
+        List<String> mutators = classView.getActiveMutators();
+
+        assertThat(mutators.size()).isEqualTo(11);
+        assertThat(mutators.get(0)).isEqualTo("CONDITIONALS_BOUNDARY");
+    }
+
+    /**
+     * Checks if the information of a mutation is displayed correctly.
+     */
+    @Test
+    public void verifyMutationInformation() {
+        MutationDetailView classView = getClassView();
+
+        MutationInformationTable informationTable = classView.getMutationInformation();
+        MutationInformationTableEntry entry = informationTable.getDataEntries().get(ROW_KEY);
+
+        assertThat(entry.getKey()).isEqualTo(ROW_KEY);
+        assertThat(entry.getMutationInformation()).contains("1.1", "Location : exists", "edu.hm.hafner.util.PathUtilTest",
+            "PathUtil::exists", "KILLED");
+    }
+
+    /**
+     * Checks if the links are set correctly. Bidirectional linking of source and information of mutation.
+     */
+    @Test
+    public void verifySourceLink() {
+        MutationDetailView classView = getClassView();
+
+        MutationSourceTable sourceTable = classView.getMutationSource();
+        MutationSourceTableEntry sourceEntry = sourceTable.getDataEntries().get(ROW_KEY);
+
+        MutationInformationTable informationTable = classView.getMutationInformation();
+        MutationInformationTableEntry informationEntry = informationTable.getDataEntries().get(ROW_KEY);
+
+        String sourceLink = extractPageAnchor(sourceEntry.getLink());
+        String mutationLink = extractPageAnchor(WebElementUtils.getAttribute(informationEntry.getClickable(),
+            WebElementUtils.LINK_ATTRIBUTE));
+
+        assertThat(sourceLink).isEqualTo(mutationLink);
+    }
+
+    /**
+     * Checks if the source area is displayed correctly.
+     */
+    @Test
+    public void verifySource() {
+        MutationDetailView classView = getClassView();
+
+        MutationSourceTable sourceTable = classView.getMutationSource();
+        MutationSourceTableEntry entry = sourceTable.getDataEntries().get(ROW_KEY);
+
+        assertThat(entry.getKey()).isEqualTo(ROW_KEY);
+        assertThat(entry.getSourceCode()).contains("return Files.exists(Paths.get(fileName));");
+        assertThat(entry.getMutationDetail()).contains("edu/hm/hafner/util/PathUtil::exists");
+        assertThat(entry.getClickable().getText()).isEqualTo("2");
```

Comment:
The mutation detail is always visible. The use CSS class `pop` indicates that it should show up as a popup. @gigaSproule Is this a known bug of the plugin?

Reply:
I'm not aware of this being a known bug, but if I'm honest, I've not used Jenkins, and by extension this plugin, in a _very_ long time, so I wouldn't commit one way or the other.

Reply:
@jwalteri Any plans to finish this PR?